db/migrate/20220807184623_remove_sent_from_invitation_status_enum.rb

```diff
@@ -3,7 +3,7 @@ def change
     reversible do |direction|
       direction.up do
         execute "UPDATE invitations SET status='pending' WHERE status='sent'"
-        remove_enum_value :invitation_status, 'sent'
+        # remove_enum_value :invitation_status, 'sent'
```

Comment:
Huh. I assume this migration had already run in on prod and this enum value exists there.

Would it make sense to also comment out the down-path, just for cleanliness sake?

And did you get a sense if we're going to have a problem with these enum migrations in the future? Or did you run some commands on the Heroku Postures instance to make them play nice going forward?

Reply:
Technicaly, I ran this migration against production, so that the enum value isn't getting removed anymore :'( ... so now this PR doesn't need to get merged and everything will keep working...

But it's probably worth fixing the actual drift; I just didn't have any enthusiasm for it beyond unfucking the deploy and wanted to move on to something fun.